doc/sphinxman/source/scf.rst

```diff
@@ -648,7 +648,9 @@ ERI Algorithms
 The key difficulty in the SCF procedure is treatment of the four-index ERI
 contributions to the Fock Matrix. A number of algorithms are available in
 |PSIfour| for these terms. The algorithm is selected by the |globals__scf_type|
-keyword, which may be one of the following
+keyword. Some such algorithms consist of a single algorithm applied to
```

Comment:
Changes in doc/* files are documentation updates regarding CompositeJK - worth looking at.